TezosKit/Models/ConseilEntity.swift

```diff
@@ -0,0 +1,10 @@
+// Copyright Keefer Taylor, 2019
+
+import Foundation
+
+/** An enum representing possible entity types on a Conseil service. */
+// TODO: Bug Mike if this list can be code-genned
```

Comment:
It cannot be generated, but it can be queried: https://github.com/Cryptonomic/Conseil/blob/master/doc/MetadataAndQuery.md#metadata-interface